.github/workflows/pull-request-ci.yaml

```diff
@@ -0,0 +1,307 @@
+        submodules: recursive
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: cache.tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/tools
+        key: ${{ runner.os }}-tools
+    - run: make tools-install
+
+    # Run the linters; split them out so it's easier to identify which one failed
+    - run: make shellcheck
+    - run: make yamllint
+    - run: make helmlint
+
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    needs: lint
+    env:
+      PINNED_TOOLS: true
+      TOOLS_DIR: ${{ github.workspace }}/tools
+    defaults:
+      run:
+        working-directory: kubecf
+    steps:
+    - name: Checkout KubeCF
+      uses: actions/checkout@v2
+      with:
+        path: kubecf
+        fetch-depth: 0
+        submodules: recursive
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: cache.tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/tools
+        key: ${{ runner.os }}-tools
+    - run: make tools-install
+    - run: make kubecf-bundle
+      env:
+        TARGET_FILE: ${{ github.workspace }}/kubecf-bundle.tgz
+    - run: |
+        mkdir bundle
+        tar xf kubecf-bundle.tgz -C bundle
+      working-directory: ${{ github.workspace }}
+    - name: Upload bundle
+      uses: actions/upload-artifact@v2
+      with:
+        name: kubecf-bundle.tgz
+        path: ${{ github.workspace }}/bundle
+
+  tests:
+    name: tests
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    needs: build
+    env:
+      # For KubeCF
+      PINNED_TOOLS: true
+      TOOLS_DIR: ${{ github.workspace }}/tools
+      FEATURE_AUTOSCALER: "true"
+      # For catapult
+      # Note that credentials-related parts are in the individual steps
+      BACKEND: gke
+      DOWNLOAD_CATAPULT_DEPS: "false"
+
+    defaults:
+      run:
+        working-directory: kubecf
+
+    strategy:
+      matrix:
+        include:
+        - backend: diego
+          experimental: false
+        - backend: eirini
+          experimental: true
+
+    steps:
+    - name: Checkout KubeCF
+      uses: actions/checkout@v2
+      with:
+        path: kubecf
+        submodules: recursive
+    - name: Checkout catapult
+      uses: actions/checkout@v2
+      with:
+        repository: SUSE/catapult
+        path: catapult
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: cache.tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/tools
+        key: ${{ runner.os }}-tools
+    - run: make tools-install
+
+    - id: download-bundle
+      name: Download chart bundle
+      uses: actions/download-artifact@v2
+      with:
+        name: kubecf-bundle.tgz
+        # The artifact upload/download creates an extra directory
+        path: ${{ github.workspace }}/kubecf-bundle
+
+    - name: Download gomplate
```

Comment:
This should be a dependency we download using `make <something>`.
@jandubois can help us do that.
Ideally, the code to download dependencies should live in `kubecf-tools`.

Reply:
It's ok not to put it in `kubecf-tools`, but please add a new tool here: https://github.com/cloudfoundry-incubator/kubecf/tree/master/scripts/tools

Reply:
Done https://github.com/cloudfoundry-incubator/kubecf/pull/1217/commits/9e0ba90cdfe9b72db1ac02f86d916c00002b384e